src/atoms/atomgraph.jl

```diff
@@ -124,6 +121,39 @@ function AtomGraph(
     return ag
 end
 
+"""
+    AtomGraph(crys::Crystal; id="", cutoff_radius = 8.0, max_num_nbr = 12, dist_decay_func = inverse_square)
+
+Construct an AtomGraph object from a Crystal object (defined in Xtals.jl). For now, only supports cutoff-based graph building.
+
+# Required Arguments
+- `crys::Crystal`: Crystal from which to build the graph
+
+# Optional Arguments
+- `id::String`: ID associated with structure (e.g. identifier from online database). Defaults to the empty string.
+- `cutoff_radius::Real = 8.0`: Longest allowable distance to a neighbor, in Angstroms
+- `max_num_nbr::Integer = 12`: Largest allowable number of neighbors
+- `dist_decay_func = inverse_square`: Function by which to assign edge weights according to distance between neighbors
+
+# Note
+`max_num_nbr` is a "soft" limit – if multiple neighbors are at the same distance, the full neighbor list may be longer.
+"""
+function AtomGraph(
```

Comment:
Can we also have a reference to this docstring in the docs?

Reply:
added it in latest commit